src/guides/v2.4/config-guide/prod/config-reference-systemxml.md

```diff
@@ -265,6 +265,21 @@ A `<field>`-Tag can have the following children:
 {:.bs-callout-warning}
 The nodes `more_url`, `demo_url`, `requires` and `options` are defined by a different core payment model and are only used once. These nodes are not reusable.
 
+### Field input type reference
+
+Each field has the default type of ´text´ to display its frontend input type. The following input field types can be found and used in the core at this time :
+
+| Field Type    | Description                                                                                                       |
+|---------------|-------------------------------------------------------------------------------------------------------------------|
+| ´text´        | Default. Standard text field.                                                                                     |
+| ´select´      | A common HTML select. Only one item can be selected.                                                              |
```

Comment:
It should also be mentioned that this covers a YES/NO and it requires a source_model to manage the items on the list

EG - Normal dropdown, may need custom `source_model` - also used for Yes/No. See `Magento\Search\Model\Adminhtml\System\Config\Source\Engine` for an example